### PR TITLE
add prompt and use mem-fs instead of fse.sync method

### LINKED ISSUE #13: Error when running the stream generator several times
When running the stream generator several times, there are duplicate key errors in the yaml file.
the placement algorithm when the key exists is to redo.

### PR DESCRIPTION
**Pull request by antarus**
Remove unused code
add code for detect property on one line
remove all fse sync code
correct unit test
fix #13 issue

### CODE REVIEW

File: nbproject/project.properties [hunk 1]
```diff
@@ -0,0 +1,3 @@
+files.encoding=UTF-8
```
Comment (pascalgrimaud) [inline nbproject/project.properties]: I don't think this file need to be committed

File: nbproject/project.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (pascalgrimaud) [inline nbproject/project.xml]: same, I don't think this file need to be committed

File: .gitignore [hunk 3]
```diff
@@ -9,3 +9,4 @@ atlassian-ide-plugin.xml
 test/temp/
 *debug.log*
 .vscode
+/nbproject/private/
```
Comment (pascalgrimaud) [inline .gitignore]: Need add:
- yarn-error.log
- /nbproject/

**Comment by antarus** [comment]
oups, sorry it's corrected.

**Comment by pascalgrimaud** [comment]
No worry, don't be sorry for that :-)

Another information: in the initial comment of this PR, if you put `fix #13` in the text, the issue related to https://github.com/hipster-labs/generator-jhipster-spring-cloud-stream/issues/13 will be closed automatically, once this PR is merged

**Pull request merged**